AzureAD: Add option to force fetch the groups from the Graph API (#56916) (#56947)
grafana/grafana: docs/sources/setup-grafana/configure-security/configure-authentication/azuread.md

> Note: The token must include the `GroupMember.Read.All` permission for group overage claim calls to succeed.
> Admin consent may be required for this permission.

### Force fetching groups from Microsoft graph API

> To force fetching groups from Microsoft Graph API instead of the `id_token`. You can use the `force_use_graph_api` config option.

Reviewer: ```suggestion
To force fetching groups from Microsoft Graph API instead of the `id_token`, use the `force_use_graph_api` configuration option.
```
